src/ai/backend/manager/registry.py

```diff
@@ -3162,9 +3167,56 @@ async def get_commit_status(
         }
 
     async def commit_session(
+        self,
+        session: SessionRow,
+        new_image_ref: ImageRef,
+        *,
+        extra_labels: dict[str, str] = {},
+    ) -> Mapping[str, Any]:
+        """
+        Commit a main kernel's container of the given session.
+        """
+
+        kernel: KernelRow = session.main_kernel
+        if kernel.status != KernelStatus.RUNNING:
+            raise InvalidAPIParameters(
+                f"Unable to commit since kernel(id: {kernel.id}) of session(id: {session.id}) is"
```

Comment:
Though this is not a strictly documented convention, I'd suggest the following to keep consistency with existing log formats:
```suggestion
                f"Unable to commit since the kernel k:{kernel.id} (of s:{session.id}) is"
```